params/config.go

```diff
@@ -139,7 +177,7 @@ type UpgradeConfig struct {
 
 // AvalancheContext provides Avalanche specific context directly into the EVM.
 type AvalancheContext struct {
-	BlockchainID common.Hash
```

Comment:
Sorry I'm just seeing this PR now, but should we leave the `BlockchainID` in the AvalancheContext here?

I think we will need it for teleporter because messages are sent to/from specific chains, rather than from subnets as we say commonly.

Reply:
we decided to remove `BlockchainID` here since that value is set through the same snow context's `ChainID` value, and since now we save the snow context in AvalancheContext, opted to remove the extra `BlockchainID` field.

Reply:
Ah gotcha, didn't realize it was available elsewhere already. Sounds good, thanks.